### DISCUSSION TITLE
Setup Auth0 as a provider

### DISCUSSION THREAD

**Discussion by ordishc** [strapi/strapi#6584] [Legacy Topics]
Hello, I am trying to add Auth0 as a provider so a user can login using Auth0 and it will generate a user within Strapi. I have got it working with other examples in purest but I cannot get it to work with Auth0.

I have followed the documentation here - https://strapi.io/documentation/v3.x/plugins/users-permissions.html#adding-a-new-provider-to-your-project 

When I hit this url - http://localhost:1337/connect/auth0 it gives me an error off `Error: Invalid URI "https:///[subdomain].auth0.com/authorize?client_id=......." This can be caused by a crappy redirection.`

Essentially I am trying to use Auth0 for security reasons as I don't really want to use the default email and password for strapi by storing the JWT in cookie or local storage.

Has anyone else managed to integrate Auth0 with Strapi? I've gone through docs in grant/purest and looked all over forums but haven't been able to find any solutions!

Thanks!

**Comment by madarche** [comment]
See #7206
